tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinatorHelpers.Core.cs

```diff
@@ -29,6 +32,65 @@ internal static void CheckAndBlock(this Security security, HttpContext context,
         }
     }
 
+    private static Dictionary<string, string[]> GetResponseHeadersForWaf(IHeaderDictionary headers)
```

Comment:
I'd think it's necessary for this PR, but I was wondering if we could share some code here with `GetBasicRequestArgsForWaf` which does something similar for the request headers.

Also, I know this is what the other method does, but the warning looks a bit strange. I shouldn't really happen, and if it does perhaps we should be merging the two list of values.

Reply:
Done!